### DISCUSSION TITLE
Return to top broken

### DISCUSSION THREAD

**Discussion by graham-rc** [janraasch/hugo-scroll#179] [General]
Return to top button seems to be broken, it links to e.g. github.site/#site-head instead of github.site/mysite/#site-head (one dir up). I'm having a look into it, but I'm at home in C++ so js/html etc is not my strong point. I'll put a PR in if I figure it out.

**Comment by graham-rc** [comment]
Sorry - I can see this has been mentioned already in issues.

**Comment by zjedi** [comment]
@graham-rc try master now, should be fixed

**Reply by graham-rc** [reply]
Looks good! Cheers.
